### ISSUE TITLE
Missing file in pecl archive

### ISSUE THREAD

**Issue by remicollet** [phpredis/phpredis#1709]
`tests/RedisSentinelTest.php` is missing in packahe.xml, making impossible to run the test suite from the pecl archive.

P.S. don't understand which branch must be used, else I will have fix it myself ;)

**Comment by yatsukhnenko** [comment]
@remicollet There is a [tag](https://github.com/phpredis/phpredis/tree/5.2.0RC1)

**Comment by remicollet** [comment]
Yes, I know there is a tag, but this is not a branch.
And not something where we can commit things in 5.2.0

**Comment by yatsukhnenko** [comment]
@michael-grunder could you re-upload 5.2.0RC2 with RedisSentinelTest or push the branch which you used while preparing release

**Comment by michael-grunder** [comment]
Be up shortly, can't believe I forgot a file 🤦‍♂  😂

**Comment by michael-grunder** [comment]
Changes consolidated into a [release-5.2.0RC2](https://github.com/phpredis/phpredis/tree/release-5.2.0RC2) branch.

**Comment by michael-grunder** [comment]
@yatsukhnenko Just let me know if everything looks to be in place.

**Comment by yatsukhnenko** [comment]
@michael-grunder thanks

**Comment by remicollet** [comment]
@michael-grunder sorry but I don't really understand how branches are managed, can you please clarify ?

**Comment by michael-grunder** [comment]
I just created the `release-5.2.0RC1` and `release-5.2.0RC2` branches to stage for the GitHub tag and PECL uploads.

After properly released I'll merge them into the develop branch.  It looks like squashing and merging `release-5.2.0RC2` into develop has confused GitHub a bit, so I backed that one out.

**Comment by michael-grunder** [comment]
I've tagged 5.2.0RC2 here and on PECL

* [GitHub](https://github.com/phpredis/phpredis/tree/5.2.0RC2)
* [PECL](https://pecl.php.net/package/redis/5.2.0RC2)

I've kept stability at `alpha` because there couldn't have been much testing since I forgot a file!

I think it might be a decent idea to add a `pear package` based test to Travis to catch any regressions like this.

Let me know if there are any issues and how the testing goes.
